locale/pl.po

```diff
@@ -6083,23 +6081,21 @@ msgstr "&OK"
 
 #: src/cloud/audiocom/ShareAudioDialog.cpp
 msgid "Public"
-msgstr ""
+msgstr "Publiczne"
 
 #: src/cloud/audiocom/ShareAudioDialog.cpp
-#, fuzzy
 msgid "Anyone will be able to listen to this audio."
 msgstr ""
-"Nie będzie można odtwarzać/nagrywać dźwięku.\n"
+"Każdy będzie mógł słuchać tego nagrania.\n"
```

Comment:
It seems like this is causing the compilaiton error: 

> D:/a/audacity/audacity/locale/pl.po:6088: 'msgid' and 'msgstr' entries do not both end with '\n'

Reply:
Done.